docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-commit.md

```diff
@@ -6,6 +6,7 @@ description: Commits all changes in a working tree to its checked out branch.
 # `git-commit`
 
 `git-commit` commits all changes in a working tree to its checked out branch.
+If there are no differences from the current `HEAD`, this step is skipped and no new commit is created.
```

Comment:
fyi... We observe a soft limit of 80 chars and a hard(er) limit of 120. It's usually best to reformat the whole paragraph. There are editor plugins that make this quick and easy. I use [rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap).